daemon/daemon_unix.go

```diff
@@ -206,6 +209,35 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *runconfig.HostC
 	return warnings, nil
 }
 
+func checkCpuset(hostConfig *runconfig.HostConfig, sysInfo *sysinfo.SysInfo) error {
+	if err := checkAvailable(hostConfig.CpusetCpus, sysInfo.Cpus); err != nil {
+		return fmt.Errorf("Cpus %s provided but not available. Available cpus are: %s.", hostConfig.CpusetCpus, sysInfo.Cpus)
+	}
+	if err := checkAvailable(hostConfig.CpusetMems, sysInfo.Mems); err != nil {
+		return fmt.Errorf("Memory nodes %s provided but not available. Available memory nodes are: %s.", hostConfig.CpusetMems, sysInfo.Mems)
+	}
+	return nil
+}
+
+// TODO(runcom): move this utility func somewhere else and unit-test it
+// tests on this are actually covered in integration-cli
+func checkAvailable(provided, available string) error {
```

Comment:
This is also a weird name, and a weird place.
Any ideas where to put this? I'd rather not add a new `TODO` here.